fastlane_bot/helpers/routehandler.py

```diff
@@ -555,6 +555,23 @@ def _get_flashloan_struct(self, trade_instructions_objects: List[TradeInstructio
 
         return flashloans
 
+    @staticmethod
+    def native_gas_token_to_wrapped(tkn: str, wrapped_address: str, native_address: str):
```

Comment:
You can make this a member method instead of a static method, i.e., take `self` as the input argument, and then:
- Instead of passing `wrapped_address` as input, you can obtain it directly from `self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS`
- Instead of passing `native_address` as input, you can obtain it directly from `self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS`

It will also make your calls to this function (further below) much cleaner and easier to read.

Reply:
I considered this but decided to make it static to be able to remove its dependency on the Config object.

Reply:
This argument doesn't make any sense whatsoever.
If you ever change or remove the Config-Object dependency in this class, then you do it in this function, and not in every other function which calls it.
As it stands, this function does nothing other than `return x if condition else y`.
It is not specific to the ETH/WETH issue other than by the names that you chose for the function itself, as well as to each one of its parameters.
In that case, you may as well move it to the general-utility module, and name it something generic...
Or alternative, simply do the `return x if condition else y` part directly where you call this function (which pretty much means reverting this change altogether, if I remember correctly).
A third option would be to pass the `self.Config` object to this method, making it indeed specific to the ETH/WETH issue (since it will then read the related attributes from that object).
But again - if you choose to implement this method **in this class**, then the most correct way to do so is by letting it handle everything (meaning - it takes `self` as input, and performs everything else internally).

Reply:
Ok, that makes sense. I implemented your suggestion of removing static from the function & using the Config object.

Reply:
Have you fixed accordingly, every place you call this function?